cypress/support/commands.js

```diff
@@ -79,3 +79,22 @@ Cypress.Commands.add('setMvtId', (str) => {
     log: true,
   });
 });
+
+Cypress.Commands.addAll({
```

Comment:
Cypress command added to simulate the client going offline. This is so we can test recaptcha verification in this scenario